auction-house/js/package.json

```diff
@@ -10,7 +10,7 @@
     "postversion": "git add package.json && git commit -m \"chore: update $npm_package_name to v$npm_package_version\" && git tag $npm_package_name@$npm_package_version",
     "prepublishOnly": "yarn check:publish-ready",
     "postpublish": "git push origin && git push origin --tags",
-    "build:docs": "typedoc",
+    "build:docs": "docsdir=../../docs/auction-house; rimraf $docsdir && typedoc --out $docsdir",
```

Comment:
Pretty sure that `env_var=` won't work on windows (in case peeps wanna run this locally).
Not a huge deal though, esp. considering we don't have many windows devs :)

Reply:
Oh, shoot - good to know! I didn't even know that about our Windows counterparts 😰 the alternative (that I know of) is to repeat the path to docs.

If you're ok with possible OS compat, I will leave as is?

Reply:
Given this runs in CI it's probably fine